src/formatDate.js

```diff
@@ -43,14 +43,34 @@
  * ) // '18.02.1997'
  *
  * @param {string} date
- * @param {string[]} fromFormat
- * @param {string[]} toFormat
+ * @param {string[]} fromForm
+ * @param {string[]} toForm
  *
  * @returns {string}
  */
 
 function formatDate(date, fromFormat, toFormat) {
```

Comment:
The function signature doesn't match with the actual function implementation. It should be 'function formatDate(date, fromForm, toForm)'